synapse/config/emailconfig.py

```diff
@@ -238,6 +240,14 @@ def read_config(self, config, **kwargs):
             self.email_add_threepid_template_success_html_content = self.read_file(
                 filepath, "email.add_threepid_template_success_html"
             )
+        elif self.account_threepid_delegate_msisdn:
```

Comment:
it feels a bit random that this is in `emailconfig.py`. wouldn't it be more appropriate to put it in `RegistrationConfig`?